app/gui/view/welcome-screen/src/lib.rs

```diff
@@ -82,19 +83,24 @@ pub struct Model {
 
 impl Model {
     /// Constructor. `frp` is used to set up event handlers on buttons.
-    pub fn new(app: &Application) -> Self {
+    pub fn new(
+        app: &Application,
+        open_project: frp::Any<String>,
+        create_project: frp::Any<Option<String>>,
+    ) -> Self {
         let application = app.clone_ref();
         let logger = Logger::new("WelcomeScreen");
         let display_object = display::object::Instance::new(&logger);
 
-        let side_menu = SideMenu::new(&logger);
-        let template_cards = TemplateCards::new(&logger);
+        let side_menu = SideMenu::new(&logger, open_project, create_project.clone_ref());
+        let template_cards = TemplateCards::new(&logger, create_project.clone_ref());
         let dom = Self::create_dom(&logger, &side_menu, &template_cards);
         display_object.add_child(&dom);
 
-        // Use `fullscreen_vis` layer to lock position when panning
+        // Use `welcome_screen` layer to lock position when panning.
+        app.display.scene().dom.layers.welcome_screen.manage(&dom);
+        // Use `panel` layer to lock zoom.
         app.display.scene().layers.panel.add_exclusive(&dom);
```

Comment:
We say in comments: once about panning, once about zooming, but actually both lines are for handling zooming and panning, aren't they?

Reply:
Oh well, actually `app.display.scene().layers.panel.add_exclusive(&dom);` does nothing. I thought it would disable zooming behavior (as you saw during the review), but it's not the case. I checked desktop IDE on Mac - zooming is not working for Welcome Screen, with or without this line. So I removed it 🤷🏻‍♂️